targets/esp8266/include/jerry_extapi.h

```diff
@@ -28,10 +28,10 @@
 
 #define JS_VALUE_TO_NUMBER(val_p) \
     ((val_p)->type == JERRY_API_DATA_TYPE_FLOAT32 ? \
-       static_cast<double>((val_p)->v_float32) : \
+       (double)((val_p)->v_float32) : \
```

Comment:
Strange indentation. Can you write this in one line or align the colon and the question mark to the same column?